openam-authentication/openam-auth-oauth2/src/main/resources/amAuthOAuth.xml

```diff
@@ -516,6 +527,18 @@ PUBLIC "=//iPlanet//Service Management Services (SMS) 1.0 DTD//EN"
                                      type="single" syntax="string" i18nKey="oidc.crypto_context_value" order="3100"
                                      resourceName="openidConnectContextValue">
                     </AttributeSchema>
+                    <AttributeSchema name="org-forgerock-auth-oauth-code-challenge-method-algorithm" 
```

Comment:
This feature will be supported from `openam-jp` edition. If I were you, I would rename the settings. (`org-forgerock-auth-oauth-code-challenge-method-algorithm` -> `jp-openam-auth-oauth-code-challenge-method`).